script.js

```diff
@@ -0,0 +1,87 @@
+const submitForm = document.querySelector('.input-box'); // 할 것 제출
+const button = document.querySelector('button'); // 입력버튼
+const textInput = document.querySelector('input'); // 추가 할 내용
+const todolist = document.querySelector('#todo-list'); // 버튼을 눌렀을 때 추가되는 곳
+const todo = document.querySelector('.todo');
+const done = document.querySelector('.done');
+const doneList = document.querySelector('#done-list');
+var count_todolist = 0;
+var count_donelist = 0;
+
+var localStorage = [];
+
+// 폼 제출 비동기 함수
+submitForm.addEventListener('submit', (e) => {
+    e.preventDefault();
+    console.log('hi');
+    if (textInput.value) {
+        // textInput.value에 내용이 담겼으면
+        const div = document.createElement('div');
+        const span = document.createElement('span');
+        span.textContent = textInput.value;
+        // div 안 span에 input 내용을 넣어줌
+        span.classList.add('content');
+        span.style.display = 'inline-block';
+        span.style.width = '250px';
+        div.append(span);
+
+        // double click로 todo -> done, done -> todo 구현하는 event를 span에 적용
+        span.addEventListener('dblclick', (e) => {
+            console.log(e.target.parentNode.id);
+            const parent = e.target.parentNode;
+            if (parent.parentNode.id === 'todo-list') {
+                doneList.append(e.target.parentNode);
+                span.style.textDecoration = 'line-through';
+                count_donelist += 1;
+                count_todolist -= 1;
+            } else {
+                todolist.append(e.target.parentNode);
+                span.style.textDecoration = 'none';
+                count_donelist -= 1;
+                count_todolist += 1;
```

Comment:
별 상관은 없지만 자바스크립트도 요런 기능이 있답니다

```suggestion
                count_donelist--;
                count_todolist++;
```

Reply:
사실 문법을 알고 있었는데요! 제가 강의 들었을 때 += 1 이런식으로 해줬을 때의 장점을 들어가지구,, 
지금은 기억이 안나네요 ㅠㅠ  한 번 찾아 보겠습니다~!

Reply:
아하 ㅎㅎ 편하신대로 하면 좋을것같아요